alma/alma.php

```diff
@@ -58,9 +58,12 @@ public function __construct()
 
         $this->limited_currencies = ['EUR'];
 
-        $version = explode('.', _PS_VERSION_);
-        $version[3] = (int) $version[3] + 1;
-        $version = implode('.', $version);
+        $version = _PS_VERSION_;
+        if (version_compare(_PS_VERSION_, '1.5.6.1', '<')) {
+            $version = explode('.', _PS_VERSION_);
+            $version[3] = (int) $version[3] + 1;
+            $version = implode('.', $version);
```

Comment:
it's discussed but personaly i love Regex ;) 
somethink like that  =>
`preg_replace_callback('/(.*)(\d)$/',function($m){return $m[1].(int)++$m[2];},'_PS_VERSION_');`

Reply:
@ClaireAlma , @Francois-Gomis 
I found this `$version = substr_replace($version, substr($version, -1) + 1, -1);`
WDYT ?

Reply:
It's seams more lisible than `preg_replace_callback`